components/locid/src/helpers.rs

```diff
@@ -11,26 +11,25 @@ use litemap::store::*;
 /// Internal: A vector that supports no-allocation, constant values if length 0 or 1.
 #[derive(Debug, Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
 pub(crate) enum ShortVec<T> {
-    Empty,
-    Single(T),
+    ZeroOne(Option<T>),
```

Comment:
Please add a comment why we're doing this.

Reply:
Added a comment.